src/components/sidebar/lineView/LineInfoTab.tsx

```diff
@@ -48,16 +49,16 @@ class LineInfoTab extends React.Component<
         super(props);
         this.state = {
             isLoading: true,
-            lineTopics: []
+            lineHeaders: []
         };
     }
 
     async componentWillMount() {
         const lineId = this.props.lineStore!.line!.id;
-        const lineTopics: ILineTopic[] = await LineTopicService.fetchLineTopicsForLineId(
+        const lineHeaders: ILineHeader[] = await LineHeaderService.fetchLineHeaders(
             lineId
         );
-        this.initLineTopicItems(lineTopics);
+        this.initLineHeaderItems(lineHeaders);
```

Comment:
Maybe unnecessary to make this in to a function of its own? Just moving around lineHeaders and not making changes to it.

Reply:
We try to keep react's lifecycle methods as clean as possible. The details of that code is not very interesting so it is better to put it in its own method